.github/workflows/create-block.yml

```diff
@@ -14,13 +14,17 @@ concurrency:
 
 jobs:
     checks:
-        name: Checks w/Node.js ${{ matrix.node }} on ${{ matrix.os }}
+        name: Checks w/Node.js ${{ matrix.node.name }} on ${{ matrix.os }}
```

Comment:
This way the job names don't change, it'll still be something like "Create Block / Checks w/Node.js 22 on macos-latest (pull_request)" and it means I won't have to change required checks in github's setting back and forth.

In future we might consider keeping this and using names like "LTS" and "Next" instead of specific version numbers.